app/managers.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self.connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self.connection.commit()
+
+    def all(self) -> list:
+        cursor = self.connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [
+            Actor(*row) for row in cursor
+        ]
+
+    def update(
+            self,
+            id_to_update: int,
+            first_name: str,
+            last_name: str
+    ) -> None:
+        self.connection.execute(
+            f"UPDATE {self.table_name} "
+            f"SET first_name=?, last_name=? "
+            f"WHERE id=?",
+            (first_name, last_name, id_to_update)
+        )
+        self.connection.commit()
+
+    def delete(self, id_to_delete: int) -> None:
+        self.connection.execute(
+            f"DELETE FROM {self.table_name} "
+            f"WHERE id=?",
```

Comment:
Using f-strings for SQL queries can lead to SQL injection vulnerabilities. It's safer to use parameterized queries, which you are already doing with placeholders (?). Ensure that the table name is not user-controlled to avoid SQL injection through the table name.